lib/Db/FailedLoginAttemptMapper.php

```diff
@@ -114,4 +114,15 @@ public function deleteSuspiciousAttemptsForUidIpCombination($uid, $ip) {
 			->andWhere($builder->expr()->eq('ip', $builder->createNamedParameter($ip)))
 			->execute();
 	}
+
+	/**
+	 * @param int $threshold
```

Comment:
please always properly comment public methods

especially missing here is what `$threshold` is and what unit it has (milliseconds ? seconds ? days ?)